freqtrade/exchange/exchange.py

```diff
@@ -1411,7 +1452,8 @@ async def _async_get_candle_history(self, pair: str, timeframe: str,
                 "Fetching pair %s, interval %s, since %s %s...",
                 pair, timeframe, since_ms, s
             )
-            params = self._ft_has.get('ohlcv_params', {})
+            # TODO-lev: Does this put price into params correctly?
+            params = self._ft_has.get('ohlcv_params', {price: price})
```

Comment:
Personally, i do have my doubts / reservations about this double-use of `params` in ccxt, as it can cause some very odd circumstances and bugs (assume an exchange uses `price` as parameter for this endpoint by "ExchangeY", and a "ccxt_price" would've made it clearer that this is only a ccxt relevant parameter), but i don't think this is the right place to discuss this - and if ccxt recommends this usage and doesn't see future problems with it - i guess i'm fine with that.